diagnostic_common_diagnostics/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ catkin_package()
 
 catkin_python_setup()
 
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+
+  # Test CPU Monitor
+  add_rostest(test/launch/test_cpu_monitor.launch)
+  add_rostest(test/launch/test_cpu_monitor_level_ok.launch)
+  add_rostest(test/launch/test_cpu_monitor_level_warn.launch)
+endif()
+
 install(PROGRAMS
```

Comment:
Should add cpu_monitor.py to the install list here so that it gets included in debs.